### DISCUSSION TITLE
AI讀心電圖

### DISCUSSION THREAD

**Discussion by zixunn** [1101-GS4719-C/Python-Community-GS4719-C#323] [General]
當民眾曾經出現胸悶、心悸、昏厥等狀況，但未曾確診，或許可以進一步利用AI與穿戴式裝置監測，將有利於早期診斷、早期用藥，避免發生心衰竭，甚至危及生命的情況，這項技術若問事感覺能拯救很多生命。
[https://health.ltn.com.tw/article/breakingnews/3363427](https://health.ltn.com.tw/article/breakingnews/3363427)

**Comment by cody910120** [comment]
再加上如果發現問題可以直接打119這樣更棒了！

**Comment by 107602505** [comment]
我覺得這是一個很棒的技術利用AI將醫療資源發揮得更好，能及早知道自己身體有無狀況避免太晚就醫而危及到生命，若成功率夠高的話未來也能減少醫師工作量、縮短民眾等待的時間。

**Comment by yt7l0** [comment]
能夠早點發現、有辦法預防是最好的

**Comment by YuHengChen54** [comment]
AI結合醫療真的很實用，畢竟健康是最重要的事
